src/CombinePDF/CombinePDF.tsx

```diff
@@ -110,6 +156,12 @@ const CombinePDF = (): JSX.Element => {
                     key={pageNumber}
                     className="shadow p-1 bg-white m-1 rounded-md overflow-hidden border-4 border-white hover:cursor-move"
                   >
+                    <div
+                      onClick={(e) => onDelete(pageNumber)}
+                      className="absolute rounded-md p-1 bg-red-600 text-white text-xs cursor-pointer"
+                    >
+                      Delete
+                    </div>
```

Comment:
Hide delete button when there's only one page left? Or maybe keep it and let people delete all the pages and start over by doing `setPdf(undefined)` above instead of that return?

```suggestion
                    {pageOrder.length > 1 && (<div
                      onClick={(e) => onDelete(pageNumber)}
                      className="absolute rounded-md p-1 bg-red-600 text-white text-xs cursor-pointer"
                    >
                      Delete
                    </div>)
                    }
```